tool/README.md

```diff
@@ -27,7 +27,7 @@ $ mvn clean install
 Issue Tracking
 --------------
 
-Bugs and features are tracked within the WildFly Elytron Tool GitHub project https://github.com/wildfly-security/wildfly-elytron-tool/issues
+Bugs and features are tracked for the WildFly Elytron Tool is through JIRA https://issues.redhat.com/secure/Dashboard.jspa
```

Comment:
@suhasid098 just a minor, this could be simplified to https://issues.redhat.com . It will then get redirected where needed

Reply:
Okay will change that, thanks!

Reply:
Or actually might be better to use https://issues.redhat.com/projects/ELY/issues address